### ISSUE TITLE
Update Nuget package

### ISSUE THREAD

**Issue by dev-masih** [AydinAdn/MediaToolkit#112]
Can you please update the NuGet package the last version is for 2016

**Comment by GlaceusFrost** [comment]
I would also like an update to the NuGet package. I was trying to use this in a project, but need the new FramesToVideo method in order to do so.

**Comment by dev-masih** [comment]
@GlaceusFrost you can use direct source from this branch
https://github.com/it14118/MediaToolkit/tree/p2
it contains the `FramesToVideo` method

**Comment by GlaceusFrost** [comment]
Yeah. That's my temporary work around. I had just already set my project up with the NuGet package, and would prefer to use NuGet rather than a downloaded copy.
